### PR TITLE
docs: Make sample commands directly copyable

### PR DESCRIPTION
**Pull request by gibson042**

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -6,16 +6,19 @@ Usage:
...
  100% |███████████████████████████████████████████████████████| (100/100, 71 it/s)        
 
 Estimated block height at Mon Aug 21 13:23:00 PDT 2023 is 11234156
 ```
 
 ## Date estimation at future height
-
 ```shell
-% go run main.go -height 11260923 -rpc https://main.rpc.agoric.net:443 -samples 1000           
+% go run main.go -height 11260923 -rpc https://main.rpc.agoric.net:443 -samples 1000
```
Comment (dckc) [inline Readme.md]: Why keep the `%` this time?

File: Readme.md [hunk 2]
```diff
@@ -27,4 +30,4 @@ Australia/NSW: Mon Aug 21 23:01:08 AEST 2023
 Asia/Istanbul: Mon Aug 21 16:01:08 +03 2023
    US/Pacific: Mon Aug 21 06:01:08 PDT 2023
    US/Eastern: Mon Aug 21 09:01:08 EDT 2023
-```
\ No newline at end of file
+```
```
Comment (dckc) [inline Readme.md]: I can't see a difference there. missing newline? surely github would make that visible

**Review by dckc — COMMENTED** [review]
I don't quite get one part

File: Readme.md (hunk 1 above)
Comment (gibson042) [inline Readme.md]:
```suggestion
go run main.go -height 11260923 -rpc https://main.rpc.agoric.net:443 -samples 1000
```

Comment (gibson042) [reply Readme.md]: I just missed it in the edit. Fixed now.

File: Readme.md (hunk 2 above)
Comment (gibson042) [reply Readme.md]: 🤷

**Review by toliaqat — APPROVED** [review]

**Pull request merged**